components/sidebar/sidebar_channel_list/sidebar_channel_list.tsx

```diff
@@ -564,7 +566,7 @@ export default class SidebarChannelList extends React.PureComponent<Props, State
                         renderTrackVertical={renderTrackVertical}
                         renderView={renderView}
                         onScroll={this.onScroll}
-                        style={{position: 'absolute'}}
+                        style={scrollbarStyles}
```

Comment:
```suggestion
                        style={({position: 'absolute'} as CSSProperties)}
```

Reply:
So the reason we changed it because on every render a new object will get created. The component will do a shallow compare with Object.is for that prop. Now because a new object was created on render its reference is changed and previous reference is lost. React will trigger a render again because of that. Globally declaring the object means object is only created once at the start of the execution and it will remain sane through out.

Reply:
yes, you are right .. I did not think about that! Nice find BTW!